spec/fixtures/files/pulfa/C0652/c0383.xml

```diff
@@ -1,7 +1,7 @@
 <c xmlns="urn:isbn:1-931666-22-9" xmlns:pulfa="http://library.princeton.edu/pulfa" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" level="file" id="C0652_c0383"><pulfa:context><pulfa:uri>https://findingaids.princeton.edu/collections/C0652/c0383</pulfa:uri><pulfa:breadcrumbs><pulfa:crumb uri="https://findingaids.princeton.edu/collections/C0652/c0375" level="series">Series 5: Audio Recordings</pulfa:crumb></pulfa:breadcrumbs><pulfa:collectionInfo><pulfa:ARK>http://arks.princeton.edu/ark:/88435/hd76s0088</pulfa:ARK><pulfa:uri>https://findingaids.princeton.edu/collections/C0652</pulfa:uri><pulfa:collection-creators><persname source="viaf" role="cre" authfilenumber="http://viaf.org/viaf/44304596">Rodriguez Monegal, Emir, 1921-1985.</persname></pulfa:collection-creators><eadid countrycode="US" mainagencycode="US-NjP" url="http://arks.princeton.edu/ark:/88435/hd76s0088" urn="ark:/88435/hd76s0088">C0652</eadid><unittitle>Emir Rodriguez Monegal Papers</unittitle><unitid type="collection" countrycode="US" repositorycode="US-NjP">C0652</unitid><unitdate normal="1941/1985" type="inclusive">1941-1985</unitdate><unitdate normal="1965/1968" type="bulk">1965-1968</unitdate><extent>11 linear feet</extent><extent unit="boxes">24 archival boxes</extent></pulfa:collectionInfo><pulfa:request type="single">C0652 item 8</pulfa:request></pulfa:context><did><unittitle>Interview: Fitas / ERM, Tape 1-2 (A8)</unittitle><unitdate normal="0001-01-01T00:00:00Z/0001-01-01T00:00:00Z">undated</unitdate><langmaterial><language langcode="spa"/><language langcode="fre"/><language langcode="por"/><language langcode="eng"/></langmaterial><physloc type="code">mss</physloc><repository id="mss"><corpname source="lcsh">Princeton University. Library. Dept. of Rare Books and Special
-               Collections</corpname><subarea>Manuscripts Division</subarea><address><addressline>One Washington Road</addressline><addressline>Princeton, New Jersey 08544 USA</addressline></address></repository><container type="item" parent="C0652_i22">8</container><physdesc><extent type="computed" unit="folders">1 folder</extent></physdesc></did><altformavail><p>32101047382484_1_a.mp3</p><p>32101047382484_2_a.mp3</p><p>32101047382492_2_a.mp3</p></altformavail><scopecontent><p>2 audiocassettes; Tape 1, Side A written description on tape: Fitas / Monegal
+               Collections</corpname><subarea>Manuscripts Division</subarea><address><addressline>One Washington Road</addressline><addressline>Princeton, New Jersey 08544 USA</addressline></address></repository><container type="item" parent="C0652_i22">8</container><physdesc><extent type="computed" unit="folders">1 folder</extent></physdesc></did><altformavail><p>32101047382492_2_a.mp3</p><p>32101047382484_1_a.mp3</p><p>32101047382484_2_a.mp3</p></altformavail><scopecontent><p>2 audiocassettes; Tape 1, Side A written description on tape: Fitas / Monegal
```

Comment:
Are there any real changes to these fixture files? One change is an added linebreak, but I can't see what the change is for the other two lines.

Reply:
Yeah, it's hard to see. The order of the altformavails are switched to make the ordering test fail.

Reply:
We had to switch the order of the barcodes to adequately test the order of the resulting resource members in one of our tests; we needed to manipulate the order in the file vs. the filesystem sort order.

Reply:
👍 OK, that makes sense and lines up with the test that uses it